src/test/java/com/google/edith/UserStatsServletTest.java

```diff
@@ -0,0 +1,81 @@
+package com.google.edith;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.edith.servlets.UserInsightsInterface;
+import com.google.edith.servlets.UserStatsServlet;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import java.io.BufferedReader;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public final class UserStatsServletTest {
+
+  private DatastoreService datastore;
+  private final LocalServiceTestHelper testHelper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private final Gson gson = new Gson();
+  private final String USER_ID = "userId";
+
+  @Before
+  public void setUp() {
+    testHelper.setUp();
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  @Test
+  public void testServletGoodInput() throws Exception {
```

Comment:
This test name is a bit vague - can we add what method is being tested and what the expected output is?

Reply:
Done.